test/base-commands/base-command.test.js

```diff
@@ -59,6 +59,19 @@ describe('base-commands', () => {
         expect(ctx.stderr).to.contain('oy!');
       });
 
+    test
+      .twilioCliEnv()
+      .stdout()
+      .do(async (ctx) => {
+        ctx.testCmd = new BaseCommand(['-o', 'json'], ctx.fakeConfig);
+        await ctx.testCmd.run();
+        await ctx.testCmd.catch(new TwilioCliError('oh no', 1, { errors: [{ message: 'oh no' }] }));
+      })
+      .exit(1)
+      .it('can correctly display error details', (ctx) => {
+        expect(ctx.stdout).to.contain(`"message": "oh no"`);
```

Comment:
Should the error go to `ctx.stderr`?

Reply:
Since we're using the output processor here, it's going to `ctx.stdout`